awx/main/utils/mem_inventory.py

```diff
@@ -54,7 +54,7 @@ def __repr__(self):
         return '<_in-memory-group_ `{}`>'.format(self.name)
 
     def add_child_group(self, group):
-        assert group.name is not 'all', 'group name is all'
```

Comment:
@AlanCoding we should really clean these up - we're running compiled byte code, and I'll these assertions get optimized away in py3.